edu.wpi.first.wpilib.plugins.java/resources/templates/examples/Gyro/src/org/usfirst/frc/team190/robot/Robot.java

```diff
@@ -1,86 +1,45 @@
-
 package $package;
 
 import edu.wpi.first.wpilibj.AnalogGyro;
-import edu.wpi.first.wpilibj.CANTalon;
-import edu.wpi.first.wpilibj.SampleRobot;
-import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.IterativeRobot;
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.RobotDrive;
 
 /**
- * This is a sample program to demonstrate how to use a gyro sensor to make a robot drive
- * straight. This program uses a joystick to drive forwards and backwards while the gyro
- * is used for direction keeping.
- *
- * WARNING: While it may look like a good choice to use for your code if you're inexperienced,
- * don't. Unless you know what you are doing, complex code will be much more difficult under
- * this system. Use IterativeRobot or Command-Based instead if you're new.
+ * This is a sample program to demonstrate how to use a gyro sensor to make a
+ * robot drive straight. This program uses a joystick to drive forwards and
+ * backwards while the gyro is used for direction keeping.
  */
-public class Robot extends SampleRobot {
-
-    final int gyroChannel = 0; //analog input
-    final int joystickChannel = 0; //usb number in DriverStation
+public class Robot extends IterativeRobot {
 
-    //channels for motors
-    final int leftMotorChannel = 1;
-    final int rightMotorChannel = 0;
-    final int leftRearMotorChannel = 3;
-    final int rightRearMotorChannel = 2;
+    private static final double kAngleSetpoint = 0.0;
+    private static final double kP = 0.005; // propotional turning constant
 
-    double angleSetpoint = 0.0;
-    final double pGain = .006; //propotional turning constant
+    // gyro calibration constant, may need to be adjusted;
+    // gyro value of 360 is set to correspond to one full revolution
+    private static final double kVoltsPerDegreePerSecond = .0128;
```

Comment:
The value should be prefixed with 0 to match the other constants in the examples.